packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -813,6 +813,202 @@ class _TextSelectionHandleOverlayState
   }
 }
 
+/// An interface to receive selection handles from [EditableTextSelectionHandlesProvider].
+///
+/// Class implements [EditableTextGestureDetectorClient] will be able to use
+/// [EditableTextSelectionHandlesProvider].
+///
+/// See also:
+///
+///  * [EditableTextSelectionHandlesProvider], which provide basic selection handles
+///    for [EditableText].
+///  * [TextField], which implements this abstract class
+///  * [CupertinoTextField], which implements this abstract class
+abstract class EditableTextGestureDetectorClient {
```

Comment:
The name of this should indicate that it belongs to EditableTextSelectionHandlesProvider.